src/cross_toml.rs

```diff
@@ -23,6 +24,10 @@ pub struct CrossBuildConfig {
     xargo: Option<bool>,
     build_std: Option<bool>,
     default_target: Option<String>,
+    #[serde(default)]
+    pre_build: Vec<String>,
```

Comment:
I think this should be similar to #801, and be optional.

Reply:
Definitely, I didn't do the change here because for it I needed to change the get function